packages/components/stories/icon/icons.js

```diff
@@ -58,15 +58,20 @@ export const icons =
     ],
     'brand': [
         'IcBrandBinarybot',
+        'IcBrandCtraderWordmark',
+        'IcBrandCtrader',
```

Comment:
These are autogenerated files do we need to add the Icon in these files

Reply:
These are added automatically when we build.